angular/src/lib/select/select.component.ts

```diff
@@ -78,6 +78,8 @@ export class SelectChange {
       <md-list
         #panel
         role='listbox'
+        class="md-list--vertical"
```

Comment:
Why is this not using the tabType prop?

Reply:
Nice, I see it now, looks like tabType prop applies this class. Switching it to prop.